packages/liveblocks-react/src/room.tsx

```diff
@@ -2376,6 +2501,49 @@ function useThreadsSuspense<M extends BaseMetadata>(
   return state;
 }
 
+/**
+ * (Private beta) Returns a history of versions of the current room.
+ *
+ * @example
+ * const { versions } = useHistoryVersions();
+ */
+function useHistoryVersionsSuspense(): HistoryVersionsStateResolved {
```

Comment:
This hook is also missing the poller that updates the content because we didn't want to have "yet another poller". While it works, it lacks a way to refetch.

If we adopt React 19's `use()`, does that also mean clients will have to be on React 19?